Policies/github-acceptable-use-policies.md

```diff
@@ -62,14 +55,33 @@ You may not scrape the Service for spamming purposes, including for the purposes
 
-#### 7. Excessive Bandwidth Use
+### 7. Excessive Bandwidth Use
+The Service's bandwidth limitations vary based on the features you use. If we determine your bandwidth usage to be significantly excessive in relation to other users of similar features, we reserve the right to suspend your Account, throttle your file hosting, or otherwise limit your activity until you can reduce your bandwidth consumption. See the [GitHub Additional Product Terms](/articles/github-additional-product-terms) for more details on specific features' bandwidth limitations.
+
+### 8. Advertising on GitHub
+**Short version:** *We do not generally prohibit use of GitHub for advertising. However, we expect our users to follow certain limitations, so GitHub does not become a spam haven. No one wants that.*
+
+While we understand that you may want to promote your Content by posting supporters' names or logos in your Account, the primary focus of the Content posted in or through your Account to the Service should not be advertising or promotional marketing. This includes Content posted in or through Pages, Packages, repositories, and all other parts of the Service. You may include static images, links, and promotional text in the README documents or project description sections associated with your Account, but they must be related to the project you are hosting on GitHub. You may not advertise in other Users' Accounts, such as by posting monetized or excessive bulk content in issues.
+
+You may not promote or distribute content or activity that is illegal or otherwise prohibited by our [Terms of Service](/github/site-policy/github-terms-of-service/), [Community Guidelines](/github/site-policy/github-community-guidelines/), or [Acceptable Use Policy](/github/site-policy/github-acceptable-use-policies/), including excessive automated bulk activity (for example, spamming), get-rich-quick schemes, and misrepresentation or deception related to your promotion.
+
+If you decide to post any promotional materials in your Account, you are solely responsible for complying with all applicable laws and regulations, including without limitation the U.S. Federal Trade Commission's Guidelines on Endorsements and Testimonials. We reserve the right to remove any promotional materials or advertisements that, in our sole discretion, violate any GitHub terms or policies.
+
+### 9. Spam and Inauthentic Activity on GitHub
+Automated excessive bulk activity and coordinated inauthentic activity, such as spamming, are prohibited on GitHub. Prohibited activities include:
+* bulk distribution of promotions and advertising prohibited by GitHub terms and policies
+* inauthentic interactions, such as fake accounts and automated inauthentic activity
+* rank abuse, such as automated starring or following
+* creation of or participation in secondary markets for the purpose of the proliferation of inauthentic activity
+* using GitHub as a platform for propagating abuse on other platforms
```

Comment:
@bluemazzoo I'm interested in hearing more about what this bullet is intended to cover/prevent. The @github/user-policy team often receives reports of "this code is being used to violate the terms of XYZ Platform," and we've always held the line that we don't have the ability to enforce third party terms. Examples include game cheats, or a script that creates spammy content and is alleged to be used in a chat application. It's often an unpopular stance, and I'm concerned this particular addition may contribute to further confusion/frustration from users who expect us to disable a repo that they allege is being used to violate terms of service on other platforms.